packages/terra-switch/tests/wdio/switch-spec.js

```diff
@@ -0,0 +1,47 @@
+Terra.describeViewports('Switch', ['tiny', 'medium', 'large'], () => {
+  const ignoredDisabledAlly = { 'color-contrast': { enabled: false } };
```

Comment:
Are there any color-contrast Axe issue? If not, then this can be removed

Reply:
Yes..! Not required with new Css values. [removed](url).